spec/formatting.md

```diff
@@ -18,3 +18,212 @@ the local variable takes precedence.
 
 It is an error for a local variable definition to
 refer to a local variable that's defined after it in the message.
+
+## Error Handling
+
+During the formatting of a message,
+various errors may be encountered.
+These are divided into the following categories:
+
+- **Syntax errors** occur when the syntax representation of a message is not well-formed.
+
+  Example invalid messages resulting in a Syntax error:
+
+  ```
+  {Missing end brace
+  ```
+
+  ```
+  {Unknown {?placeholder}}
```

Comment:
Yes, because the `{?placeholder}` does not match any construct of the [EBNF](https://github.com/unicode-org/message-format-wg/blob/main/spec/message.ebnf). It's not Text because the `{` and `}` are not allowed, and it's not a Placeholder because those may only contain an Expression, Markup, or MarkupEnd, which in total must start with one of `$`, `:`, `+`, or `-`.

Reply:
The blindness is terrible... I read `?` as `$` 😜

"Nevermind"